src/WarcraftLegacies.Source/Quests/BlackEmpire/QuestMawofShuma.cs

```diff
@@ -0,0 +1,42 @@
+﻿using MacroTools.FactionSystem;
+using MacroTools.LegendSystem;
+using MacroTools.ObjectiveSystem.Objectives.ControlPointBased;
+using MacroTools.ObjectiveSystem.Objectives.LegendBased;
+using MacroTools.ObjectiveSystem.Objectives.QuestBased;
+using MacroTools.ObjectiveSystem.Objectives.UnitBased;
+using MacroTools.QuestSystem;
+using WarcraftLegacies.Source.Quests.Cthun;
+
+namespace WarcraftLegacies.Source.Quests.BlackEmpire
+{
+  /// <summary>
+  /// Gain an additional Forgotten one
+  /// </summary>
+  public sealed class QuestMawofShuma : QuestData
+  {
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="QuestMawofShuma"/> class.
+    /// </summary>
+    public QuestMawofShuma(LegendaryHero yorsahj, Faction ahnqiraj) : base("Maw of Shu'ma",
+      "Yor'sahj needs souls to awaken the Maw of Shu'ma for me, a terrible Forgotten One. C'thun has been gathering samples of living things for his experiments, but i can use their souls for my own purposes.",
+      @"ReplaceableTextures\CommandButtons\BTNFacelessOneWidow.blp")
+    {
+      AddObjective(new ObjectiveFactionQuestComplete(ahnqiraj.GetQuestByType<QuestMockeryOfLife>(), ahnqiraj));
```

Comment:
This doesn't really work right? Mockery of Life simply requires killing a certain number of units.... But as we discussed, Ahn'qiraj still gets credit for Mockery of Life when his allies (like Black Empire) kill units. So this objective for clarity should just say "kill x units".

Reply:
Can you see it working with another quest from Cthun?

Reply:
Nah nothing off the top of my head. First thought is that you could just use the "kill x objective" and reflavour it so it's not connected to C'thun - you'd have the exact same thing you have now but readable.